### PR TITLE
Adding in matomo configuration to replace piwik

### PR DESCRIPTION
**Pull request by Tiki-God**
Adding in Matomo to replace the outdated Piwik installation.

### CODE REVIEW

File: ansible/group_vars/hub.yml [hunk 1]
```diff
@@ -34,7 +34,10 @@ hub_containers:
     name: hub-fe
   - image: mojdigitalstudio/digital-hub-node
     name: hub-node
-
+  - image: matomo:3-apache
+    name: hub-matomo
+  - image: mojdigitalstudio/digital-hub-db
```
Comment (jhackettpps) [inline ansible/group_vars/hub.yml]: Is this the same image as mariadb?

File: docker-compose.yml [hunk 2]
```diff
@@ -52,7 +52,7 @@ services:
         hub-be:
                 image: mojdigitalstudio/digital-hub-be
                 depends_on:
-                        - hub-db
+                        2- hub-db
```
Comment (jhackettpps) [inline docker-compose.yml]: ..typo?

Comment (Tiki-God) [reply docker-compose.yml]: Bah thought I'd removed that typo!

File: ansible/group_vars/hub.yml (hunk 1 above)
Comment (Tiki-God) [reply ansible/group_vars/hub.yml]: It's the same image, but I was aiming to keep the databases separate as per current prod environment.

Comment (jhackettpps) [reply ansible/group_vars/hub.yml]: OK

**Comment by jhackettpps** [comment]
LGTM

**Pull request merged**